### ISSUE TITLE
Writeable doesn't accept chunk in end method

### ISSUE THREAD

**Issue by focusaurus** [jsantell/streambuffers#1]
```js
const streambuffers = require("streambuffers");
const out = new streambuffers.Writable();
out.end("foo");
console.log(out.toBuffer().toString());
```

I expect to be able to pass a chunk to `end` using [this api](https://nodejs.org/docs/latest/api/stream.html#stream_writable_end_chunk_encoding_callback), but it seems to chunk is ignored and the stream remains empty.

**Comment by jsantell** [comment]
Easy fix!

**Comment by jsantell** [comment]
Available via 1.0.1
